tests/snappi_tests/intf_utils/intf_accept_metrics.py

```diff
@@ -0,0 +1,89 @@
+# This file defines the interfaces that snappi tests accept external metrics.
```

Comment:
the definitions of the metric names and meta are missing in the file, we need to get them defined and show a unified format. this will be used for crafting the dashboards.